src/app/Scenes/SavedSearchAlert/SavedSearchStore.tsx

```diff
@@ -76,7 +76,11 @@ export const savedSearchModel: SavedSearchModel = {
 
   clearAllAttributesAction: action((state) => {
     if (Object.values(state.attributes).length !== 0) {
-      state.attributes = {}
+      state.attributes = {
+        // Keep only selected artist IDs
+        artistIDs: state.attributes.artistIDs,
+        artistID: state.attributes.artistID,
+      }
```

Comment:
https://lodash.com/docs/4.17.15#pick

```suggestion
      state.attributes = pick(state.attributes, ['artistIDs', 'artistID']);
```

Reply:
Does the check `if (Object.values(state.attributes).length !== 0)` still make sense here ('artistIDs' or 'artistID' will is present in most cases, right)?

Reply:
true, we can avoid this and use isEqual to set as `dirty`